### ISSUE TITLE
Doc: Add missing images to ENV_SETUP_INSTRUCTION.md on Search_path section 6.

### ISSUE THREAD

**Issue by mtreacy002** [anitab-org/bridge-in-tech-backend#70]
## Description
As a contributor,
I need to be able to see visual examples on how to setup my environment, in particular to set search_path to second bitschema,
so that I can setup BridgeInTech backend development successfully.

## Mocks
The image below shows the missing images on BIT Development Environment Setup Instructions wili that this issue is targeting.

<img width="654" alt="Screen Shot 2020-06-28 at 3 57 53 pm" src="https://user-images.githubusercontent.com/29667122/85939445-024c3c80-b959-11ea-853e-299b756cfa19.png">

### Update [Required]
- [ ] provide the images described in the `TO DO` on the wiki page mentioned above. The changes is to be placed inside the ENV_SETUP_INSTRUCTION.md inside the .github folder. 
Lines to place the related TO DO:
    * below line 69
    * below line 72
    * below line 74

### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

**Comment by mtreacy002** [comment]
Update @anitab-org/bridgeintech-maintainers . This issue is to add images on the set search_path section I've just added to Wiki BIT Development Environment Setup Instructions.

**Comment by Aaishpra** [comment]
@mtreacy002 can I work on this?

**Comment by mtreacy002** [comment]
@Aaishpra , I already assign you to issue #90. You can complete that one first before moving on to the next issue 😉

**Comment by Aaishpra** [comment]
Sure

**Comment by aastha985** [comment]
I would like to work on this.

**Comment by mtreacy002** [comment]
Sure, @aastha985. I'll assign you to this.

**Comment by mtreacy002** [comment]
Wait, sorry, @aastha985 . The issue is laabeled `First-Timers-Only`. Since you already done your first contribution to the project, you should pick another issue that is not First-Timers-Only 😉

**Comment by mtreacy002** [comment]
Let me check to see if your last issue is in frontend repo or backend. If you only have done frontend but not backend, I think then you should be allowed to take up this issue. @anitab-org/coding-team , can you please confirm this? I'm new being a maintainer 😁

**Comment by mtreacy002** [comment]
Ok, I've checked and confirmed that you've only done frontend. So, hopefully this means you can take up the issue. Am I right, @anitab-org/coding-team ?

**Comment by mtreacy002** [comment]
@aastha985 , I'm pretty sure I can assign you to this since this will be your first time contribution to frontend repo. Thanks again for picking up the issue 😉.

**Comment by aastha985** [comment]
Great! Thanks 😊 @mtreacy002

**Comment by aastha985** [comment]
@mtreacy002 In the line,
`3. Finally, run the same command on step 2 to check if the new path has been set.`
Should this be step 1 instead of step 2 ?
Output for step2
<img width="576" alt="image3" src="https://user-images.githubusercontent.com/55681187/93444434-3434c580-f8ee-11ea-8bcd-c15994edf56a.PNG">
Output for step1
<img width="309" alt="image4" src="https://user-images.githubusercontent.com/55681187/93444486-47479580-f8ee-11ea-9420-c77047ee9ecc.PNG">

**Comment by aastha985** [comment]
Also, I am getting
` -- No SQL could be generated for the selected object. `

What could possibly be the issue? Thanks in advance!
![image](https://user-images.githubusercontent.com/55681187/93448948-cab6b600-f8f1-11ea-8f59-bf83f292f501.png)

**Comment by mtreacy002** [comment]
> @mtreacy002 In the line,
> `3. Finally, run the same command on step 2 to check if the new path has been set.`
> Should this be step 1 instead of step 2 ?

You are absolutely right 😂😂. It should be step 1. Great pick up 👍

> Output for step2
> <img alt="image3" width="576" src="https://user-images.githubusercontent.com/55681187/93444434-3434c580-f8ee-11ea-8bcd-c15994edf56a.PNG">
Yep, that's about right. 

> Output for step1
> <img alt="image4" width="309" src="https://user-images.githubusercontent.com/55681187/93444486-47479580-f8ee-11ea-9420-c77047ee9ecc.PNG">

This happens coz you already done step 2. For a new contributor who only just starting the setting up process, when the database still in its initial stage, output of step 1 should be only `public` and no `bit_schema`. The picture you have above should've been the output for step 3, after the search_path is altered by step 2.

**Comment by mtreacy002** [comment]
> Also, I am getting
> `-- No SQL could be generated for the selected object.`
> 
> What could possibly be the issue? Thanks in advance!
> ![image](https://user-images.githubusercontent.com/55681187/93448948-cab6b600-f8f1-11ea-8f59-bf83f292f501.png)

Have you tried running the application by executing `python run.py` after altering the search_path? Coz database will only be created when you click the link to the app url given when you run that command (so, not just run the command, but click on the link)

**Comment by mtreacy002** [comment]
When you click on the link below
<img width="731" alt="Screen Shot 2020-09-18 at 10 20 39 am" src="https://user-images.githubusercontent.com/29667122/93541034-c1345900-f998-11ea-8c2a-07686bceca81.png">

and see the Swagger UI displayed properly
<img width="1680" alt="Screen Shot 2020-09-18 at 10 20 45 am" src="https://user-images.githubusercontent.com/29667122/93541094-dd37fa80-f998-11ea-8282-be3ad5df0865.png">

Then your tables should be created.

**Comment by aastha985** [comment]
> > @mtreacy002 In the line,
> > `3. Finally, run the same command on step 2 to check if the new path has been set.`
> > Should this be step 1 instead of step 2 ?
> 
> You are absolutely right 😂😂. It should be step 1. Great pick up 👍
> 
> > Output for step2
> > <img alt="image3" width="576" src="https://user-images.githubusercontent.com/55681187/93444434-3434c580-f8ee-11ea-8bcd-c15994edf56a.PNG">
> > Yep, that's about right.
> 
> > Output for step1
> > <img alt="image4" width="309" src="https://user-images.githubusercontent.com/55681187/93444486-47479580-f8ee-11ea-9420-c77047ee9ecc.PNG">
> 
> This happens coz you already done step 2. For a new contributor who only just starting the setting up process, when the database still in its initial stage, output of step 1 should be only `public` and no `bit_schema`. The picture you have above should've been the output for step 3, after the search_path is altered by step 2.

Thanks for the clarification😊

**Comment by aastha985** [comment]
I see this instead after clicking on the link


![image](https://user-images.githubusercontent.com/55681187/93559286-0195d600-f99d-11ea-9d63-9d5a6ceaf7e9.png)

**Comment by mtreacy002** [comment]
@aastha985 , can you show we what you put in .env (remove/hide security related vars such as password and email)? And also pls tell me your machine (OS, version and what terminal you're using, plus python version)

**Comment by aastha985** [comment]
Oh Okay, I am using windows10,git bash and Python 3.6 but I haven't created the .env file yet as I was on step 6. Maybe we can update the wiki to mention the line, `you should see that the tables are created under each schemas` after step 7. 
Thanks for the help:) @mtreacy002

**Comment by mtreacy002** [comment]
> Maybe we can update the wiki to mention the line, `you should see that the tables are created under each schemas` after step 7. 

@aastha985 , that makes sense 

> How can I send a PR for a wiki?

 I'm planning to put the link to this file on the wiki sidebar, instead of writing the content direectly as a markdown page. For now, you could copy pasted the whole page on wiki setup instruction to this doc and add your section 6 search_path on top of it, then move the line `you should see that the tables are created under each schemas` after step 7, if you want.

**Comment by mtreacy002** [comment]
Another option, @aastha985 , instead of moving that line to after step 7, keep it in section 6 where it is now but add brackets like below
`Now when you run the application using python run.py (after first completing step 7) from the terminal, you should see that the tables are created under each schemas.`

Your call 😉, as long as it makes sense in terms of instruction flow.

**Comment by aastha985** [comment]
Yeah this can be done!

**Comment by mtreacy002** [comment]
> Yeah this can be done!

So, which one you're thinking of doing, if I may know? 😁

**Comment by aastha985** [comment]
2nd :)
